columnflow/tasks/framework/base.py

```diff
@@ -448,6 +448,35 @@ def get_array_function_kwargs(cls, task=None, **params):
 
         return kwargs
 
+    @classmethod
+    def resolve_config_parameter(
+        cls,
+        config_inst: od.Config,
+        param: tuple[str] | None = None,
+        param_name: str | None = None,
+    ) -> tuple[str]:
+        """
+        Resolves tuple of inputs *param* by using groups defined in the *config_inst*
+        that map a string to a tuple of strings. If *param* is empty, the default
+        defined in the config is returned.
+        """
+        if param and param != law.NO_STR:
+            param = law.util.make_list(param)
+
+            # expand to param group
+            param_group = config_inst.x(f"{param_name}_groups", {})
+            if param_group:
+                outp = []
+                for val in param:
+                    outp.extend(param_group.get(val, [val]))
+            return tuple(outp)
```

Comment:
Mh, `outp` will not always be defined here. I guess you want to return and empty tuple in this case?